templates/about.html

```diff
@@ -1,113 +1,165 @@
-      <br><span>Join us now on GitHub!</span><br>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=watch&count=true&v=2" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=star&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=new-website&type=star&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=fork&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      </p><p>
-      <span>Supports <a href="https://http2.github.io" target="_blank"><strong>http2.0</strong>/</a>http/https/<a href="https://en.wikipedia.org/wiki/SPDY" target="_blank">spdy</a>, we encourage always using <strong>https</strong>!
-      <br /><span>Currently, CDNJS is one of the most famous free and public web front-end CDN services which is used by ~2,999,000 websites worldwide, see more detail on <a href="https://trends.builtwith.com/cdn/CDN-JS" target="_blank">BuiltWith</a> and <a target="_blank" href="https://w3techs.com/technologies/overview/content_delivery/all">W3Techs</a>.</span>
-    </p>
-
-    <div class="logged-in" style="display: none;">
+        Everyone loves the&nbsp;<a target="_blank" href="https://developers.google.com/speed/libraries/?csw=1">Google
+        CDN</a>&nbsp;right? Even&nbsp;<a href="https://www.asp.net/ajax/cdn" target="_blank">Microsoft runs a public CDN</a>.
+        <br/>The problem is, they only host the most popular libraries.
+        <br/>We host all the popular libraries - JavaScript, CSS, SWF, images, etc!
+        <br/>Join us now on GitHub!
+        <br/>
+        <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=watch&count=true&v=2" frameborder="0"
+                scrolling="0" width="105px" height="20px"></iframe>
+        <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=star&count=true" frameborder="0"
+                scrolling="0" width="105px" height="20px"></iframe>
+        <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=new-website&type=star&count=true"
+                frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
+        <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=fork&count=true" frameborder="0"
+                scrolling="0" width="105px" height="20px"></iframe>
+      </p>
       <p>
-        <img id="gravatar" src="" style="float: left; width: 64px; height: 64px; border: 1px solid #ccc; margin-right: 10px;" />Hey
-        <span style="font-weight: bold;" id="name"></span>
+        Supports <a href="https://en.wikipedia.org/wiki/HTTP/2" target="_blank"><strong>http2.0</strong>/</a>http/https/<a
```

Comment:
any idea about use wiki instead of the page by IETF HTTP WG?

Reply:
This is using the wiki page, would you rather the IETF WG page? @PeterDaveHello

Reply:
Just want to know more reason about change it, no mind.